k-distribution/src/main/scripts/bin/kx

```diff
@@ -403,36 +407,46 @@ if [ "$outputFile" = "-" ]; then
   outputFile=/dev/stdout
 fi
 
+# verify and parse config variables/cmd line parameters
 if $term; then
   if [ -z "${parser_PGM+unset}" ]; then
     execute kast -d "$dir" -m "$mainModuleName" "$config_var_PGM" -o kore > "$input_file"
   else
     execute $parser_PGM "$config_var_PGM" > "$input_file"
   fi
 else
+  for configVar in `find $kompiledDir -type f -name 'sort_*' | sed 's|.*sort_\(.*\).txt|\1|'`; do
```

Comment:
let's not use find for this. Find is recursive and likely to take longer when you can probably just use a glob.